website/templates/profile/account.mako

```diff
@@ -5,39 +5,101 @@
+                        <li><a href="${ web_url_for('user_addons') }">Configure Add-ons</a></li>
+                        <li><a href="${ web_url_for('user_notifications') }">Notifications</a></li>
+                    </ul>
+                </div>
             </div>
-        </div>
-        <div class="col-md-6">
-            <div id="changePassword" class="panel panel-default">
-                <div class="panel-heading"><h3 class="panel-title">Change Password</h3></div>
-                <div class="panel-body">
-                    <form id="changePasswordForm" role="form" action="${ web_url_for('user_account_password') }" method="post">
-                        <div class="form-group">
-                            <label for="old-password">Old password</label>
-                            <input type="password" class="form-control" name="old_password">
-                        </div>
-                        <div class="form-group">
-                            <label for="password">New password</label>
-                            <input type="password" class="form-control" name="new_password">
-                        </div>
-                        <div class="form-group">
-                            <label for="password2">Confirm new password</label>
-                            <input type="password" class="form-control" name="confirm_password">
-                        </div>
-                        <button type="submit" class="btn btn-default">Update password</button>
-                    </form>
+            <div class="col-md-6">
+                <div id="connectedEmails" class="panel panel-default">
+                    <div class="panel-heading"><h3 class="panel-title">Connected Emails</h3></div>
+                    <div class="panel-body">
+                        <table class="table">
+                            <thead>
+                                <tr>
+                                    <th colspan="2">Primary Email</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <td>{{ profile().primaryEmail().address }}</td>
+                                    <td></td>
+                                </tr>
+                            </tbody>
+                        </table>
+
+                        <table class="table">
+                            <thead>
+                                <tr>
+                                    <th colspan="3">Alternate Emails</th>
+                                </tr>
+                            </thead>
+                            <tbody data-bind="foreach: profile().alternateEmails()">
+                                <tr>
+                                    <td style="width:100%">{{ $data.address }}</td>
+                                    <td><a data-bind="click: $parent.makeEmailPrimary">make&nbsp;primary</a></td>
```

Comment:
I have no strong preference. It looked odd to me to capitalize "Make" since it isn't a sentence, and capitalizing "Primary" makes it look like a header and draw too much attention.